common/fe-ste-models.go

```diff
@@ -1368,3 +1368,94 @@ func (p PreservePermissionsOption) IsTruthy() bool {
 		panic("unknown permissions option")
 	}
 }
+
+// CpkScopeInfo specifies the name of the encryption scope to use to encrypt the data provided in the request.
+// If not specified, encryption is performed with the default account encryption scope.
+// For more information, see Encryption at Rest for Azure Storage Services.
+type CpkScopeInfo struct {
+	EncryptionScope *string
+}
+
+func (csi CpkScopeInfo) Marshal() (string, error) {
+	result, err := json.Marshal(csi)
+	if err != nil {
+		return "", err
+	}
+	return string(result), nil
+}
+
+//func UnmarshalToCpkScopeInfo(cpkScopeInfoStr string) (CpkScopeInfo, error) {
```

Comment:
Check if this commented code can be removed

Reply:
Can I remove it later once CPK goes live? I want to keep it for future reference.